tests/.pest/snapshots/Countries/JordanTest/it_can_calculate_jordanian_holidays.snap

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "name": "New Year's Day",
```

Comment:
I assume there are more holidays in Jordan?

Reply:
Other holidays are calculated based on the lunar calendar.
Which we convert to gregorian calendar when needed.
Actually, many countries have holidays based on this type of calendar; therefore, we might need to update the core of the app to support it.

Reply:
You can look at Turkey: https://github.com/spatie/holidays/blob/main/src/Countries/Turkey.php#L11